scripts/dts/gen_defines.py

```diff
@@ -68,7 +68,7 @@ def main():
                 continue
 
             out_comment("Device tree node: " + dev.path)
-            out_comment("Binding (compatible = {}): {}".format(
+            out_comment("DTS Binding (compatible = {}): {}".format(
```

Comment:
Remembered that `DTS` is really the name of the `.dts` source format, so maybe just "Binding" is fine.